stores/hostdb_test.go

```diff
@@ -136,6 +136,9 @@ func TestSQLHostDB(t *testing.T) {
 		AppliedBlocks: []stypes.Block{{}},
 		AppliedDiffs:  []modules.ConsensusChangeDiffs{{}},
 	})
+	if err := ss.applyUpdates(true); err != nil {
```

Comment:
What is this about?

Reply:
I removed the sleep above but it got reverted or I forget to "pick" the change or something, thanks for catching that.